Assets/Scripts/Characters/Human/Hero.cs

```diff
@@ -1098,6 +1100,14 @@ private void AddRightForce()
         baseRigidBody.AddForce(baseRigidBody.velocity * 0.00f, ForceMode.Acceleration);
     }
 
+    private void Update(){
```

Comment:
Code looks alright, but please apply C# syntax style. This looks more Java than C#. Pressing Ctrl + K, D, should do this automatically